src/HealthChecks.AzureServiceBus/AzureEventHubHealthCheck.cs

```diff
@@ -13,50 +13,78 @@ public class AzureEventHubHealthCheck
             {
                 throw new ArgumentNullException(nameof(eventHubName));
             }
 
-            _eventHubConnectionString = connectionString.Contains(EntityPathSegment) ? connectionString : $"{connectionString};{EntityPathSegment}{eventHubName}";
-        }
+            _eventHubConnectionString = connectionString.Contains(EntityPathSegment)
+                ? connectionString
+                : $"{connectionString};{EntityPathSegment}{eventHubName}";
+        }
 
         public AzureEventHubHealthCheck(EventHubConnection connection)
         {
             if (connection == null)
             {
                 throw new ArgumentNullException(nameof(connection));
-            }
+            }
 
-            _eventHubConnectionString = $"{connection.FullyQualifiedNamespace};{EntityPathSegment}{connection.EventHubName}";
-            if(!_eventHubConnections.TryAdd(_eventHubConnectionString, new EventHubProducerClient(connection)))
-            {
-                throw new InvalidOperationException("EventHubProducerClient can't be created using the specified connection.");
+            _eventHubConnectionString =
+                $"{connection.FullyQualifiedNamespace};{EntityPathSegment}{connection.EventHubName}";
+            if (!_eventHubConnections.TryAdd(_eventHubConnectionString, new EventHubProducerClient(connection)))
+            {
+                throw new InvalidOperationException(
+                    "EventHubProducerClient can't be created using the specified connection.");
             }
         }
 
-        public async Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context, CancellationToken cancellationToken = default)
+        public async Task<HealthCheckResult> CheckHealthAsync(
+            HealthCheckContext context,
+            CancellationToken cancellationToken = default
+        )
         {
             try
             {
-                if (!_eventHubConnections.TryGetValue(_eventHubConnectionString, out var producer))
-                {
-                    producer = new EventHubProducerClient(_eventHubConnectionString);
-
-                    if (!_eventHubConnections.TryAdd(_eventHubConnectionString, producer))
-                    {
-                        return new HealthCheckResult(context.Registration.FailureStatus, description: "EventHubProducerClient can't be added into dictionary.");
-                    }
+                if (!_eventHubConnections.TryGetValue(_eventHubConnectionString, out var producer))
+                {
+                    producer = new EventHubProducerClient(_eventHubConnectionString);
+
+                    if (!_eventHubConnections.TryAdd(_eventHubConnectionString, producer))
+                    {
+                        return new HealthCheckResult(context.Registration.FailureStatus,
+                            description: "EventHubProducerClient can't be added into dictionary.");
+                    }
```

Comment:
Your PR contains a lot of not relevant changes. For example, what exactly changed in this code block?

Reply:
It's just whitespace from autoformatting apologies. I can revert these changes if you'd prefer?

Reply:
It does not matter to me, ask the maintainers.